react/features/filmstrip/components/native/Thumbnail.js

```diff
@@ -163,9 +163,11 @@ function Thumbnail(props: Props) {
             return null;
         }
 
-        dispatch(openDialog(RemoteVideoMenu, {
-            participant
-        }));
+        if (!participant.local && !participant.isFakeParticipant) {
```

Comment:
this should just be an `else` branch, since you are already doing the checks above. Also, what's with that `return null` a couple of lines above? I guess we don't need to return anything in any case, right?